src/replicate/getDocs.js

```diff
@@ -53,7 +54,11 @@ function getDocs(src, diffs, state) {
         bulkGetInfo.docs.forEach(function (doc) {
           if (doc.ok) {
             resultDocs.push(doc.ok);
+          } else if(typeof doc.error !== 'undefined') {
```

Comment:
`(!doc.error)` is fine here, also typically we do `if (` instead of `if(` (but no biggie)